js/index.js

```diff
@@ -1,68 +1,66 @@
-const bookSection = document.querySelector('.books');
-const bookForm = document.querySelector('.new_book');
-const bookAuthorInput = document.querySelector('#book-author');
-const bookTitleInput = document.querySelector('#book-title');
+class BookList {
+  constructor() {
+    this.books = JSON.parse(localStorage.getItem('books')) || [];
+    this.bookSection = document.querySelector('.books');
+    this.bookForm = document.querySelector('.new_book');
+    this.bookAuthorInput = document.querySelector('#book-author');
+    this.bookTitleInput = document.querySelector('#book-title');
```

Comment:
[OPTIONAL]
- [ ] It would be better to define these variables outside of the class, since you will not need them while instantiating the book object. please remember this is supposed to refactor your code base from the previous project.
- [ ] Also it would be better to not include the `HTML` logic in your class, since the class is only a syntactical sugar to create objects. means in the class you may only consider putting the logic of creating, removing, and accessing the books.

You may consider referring to this code snippet:
```js
class Books {
  constructor() {
    this.books = [];
  }

  addBook(title, author) {
    this.books = this.books.concat({ title, author});
    return this.books;
  }

  removeBook(bookTitle) {
    this.books = this.books.filter(({ title }) => bookTitle !== title);
    return this.books; 
  }

  // getter method
  getBooks() {
    return this.books;
  }
}
```